packages/services/src/validateContext.ts

```diff
@@ -60,6 +62,7 @@ export const validateContextTransformationEvent = (
   if (_.isEmpty(context.vckit.vckitAPIUrl)) return error('Invalid vckitAPIUrl');
   if (_.isEmpty(context.vckit.issuer)) return error('Invalid issuer');
 
+  if (_.isEmpty(context.dpp)) return error('Invalid dpp context');
   if (_.isEmpty(context.dpp.context)) return error('Invalid dpp context');
   if (_.isEmpty(context.dpp.type)) return error('Invalid dpp type');
   if (_.isEmpty(context.dpp.dlrLinkTitle)) return error('Invalid dpp dlrLinkTitle');
```

Comment:
I think the `validateContextTransformationEvent` function should include a check/validate to ensure that the `context.dppCredentials` value exists.